src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/LifeCycle/LifeCycleEffectRouter.ios.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Linq;
+using Xamarin.CommunityToolkit.Effects;
+using Xamarin.CommunityToolkit.iOS.Effects;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.iOS;
+
+[assembly: ExportEffect(typeof(LifeCycleEffectRouter), nameof(LifecycleEffect))]
+
+namespace Xamarin.CommunityToolkit.iOS.Effects
+{
+	public class LifeCycleEffectRouter : PlatformEffect
+	{
+		LifecycleEffect lifeCycleEffect;
+
+		protected override void OnAttached()
+		{
+			lifeCycleEffect = Element.Effects.OfType<LifecycleEffect>().FirstOrDefault() ??
+				throw new ArgumentNullException($"The effect {nameof(LifecycleEffect)} can't be null.");
+
+			Element.PropertyChanged += OnPropertyChanged;
```

Comment:
On iOS, AFAIK doesn't exist a way to know the UIView lifecycle, that way I need to use the Renderer property. It still needs to be an effect, because I need to access the native renderer.